deb-package-builder/extensions/mongo/debian/copyright

```diff
@@ -0,0 +1,66 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: mongo
+Upstream-Contact: Daniel Holmes <dan@djcentric.com>
+Source: https://pecl.php.net/package/mongo
+
+Files: *
+Copyright: Copyright 2009-2014 MongoDB, Inc.
+License: APACHE-2.0
+  Apache License
```

Comment:
For well-know license types, we don't need to include the text of the license. See: the example from the debian files below the main source in this same file.